main.go

```diff
@@ -1,16 +1,209 @@
 package main
 
 import (
+	"bytes"
+	"embed"
+	"html/template"
+	"io/fs"
+	"io/ioutil"
 	"log"
+	"net/http"
 	"os"
+	"strings"
 
 	"github.com/apex/gateway"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/gin-contrib/static"
 	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+	"github.com/markbates/pkger"
 )
 
+type ContactDetails struct {
+	Name 				string 				`json:"name"`
+	DesignationValue 	string 				`json:"designation_value"`
+	Business 			string 				`json:"business"`
+	CurrentWebsiteUrl 	map[string]string 	`json:"current_website_url"`
+	HelpValue 			string 				`json:"help_value"`
+	Reason 				string 				`json:"reason"`
+	Email 				string 				`json:"email"`
+	Message 			string 				`json:"message"`
+	ContactType			string 				`json:"contact_type"`
+}
+
+type ContactRepository struct {
+	HtmlTemplate *template.Template
+}
+
+const CHARSET = "utf-8"
+const SENDER = "contact@canopas.com"
+const RECEIVER = "hr@canopas.com"
+
+func New() *ContactRepository {
+	emailHtml, err := pkger.Open("/templates/email-template.html")
+	if err != nil {
+		panic(err)
+	}
+
+	emailFile, err := ioutil.ReadAll(emailHtml)
+	if err != nil {
+		panic(err)
+	}
+
+	emailTemplate := template.Must(template.New("email-template.html").Parse(string(emailFile)))
+
+	return &ContactRepository{ HtmlTemplate: emailTemplate}
+}
+
+func (repository *ContactRepository) contactDetail(c *gin.Context) {
```

Comment:
I think we have to separate code in the `contact` directory, Instead of all in the main.go file